official/utils/logs/logger.py

```diff
@@ -44,6 +44,8 @@ def __init__(self, logging_dir):
     if not tf.gfile.IsDirectory(self._logging_dir):
       tf.gfile.MakeDirs(self._logging_dir)
 
+    self.run_date = datetime.datetime.now()
```

Comment:
I think its better to have a new method to log the run_date, since its not necessary true that the training start when the logger instance is created.